### PR TITLE
Fix package counts in README

### LINKED ISSUE #369: Figure out why total download counts are not included in the README table
Instead, `{emmeans}` download counts are included:

Screenshot 2023-07-11 at 11 21 53

To reproduce, just knit the README.

### PR DESCRIPTION
**Pull request by etiennebacher**
Close #369

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -1,460 +1,461 @@
...
+library(see)
+library(datawizard)
+
+# tidverse packages
+library(lubridate)
+library(ggplot2)
+
+pkgsnms <- c("bayestestR", "correlation", "datawizard", "easystats", 
+        "effectsize", "insight", "modelbased", "performance", 
+        "parameters", "report", "see")
```
Comment (etiennebacher) [inline README.Rmd:181]: I don't know why all of these changes appear, this is the only thing I modified

Comment (IndrajeetPatil) [reply README.Rmd:181]: A lot of these are whitespace changes.

Comment (IndrajeetPatil) [inline README.Rmd:181]: Maybe use `easystats:::.packages_on_cran()` instead?

Comment (etiennebacher) [reply README.Rmd]: done

**Review by IndrajeetPatil — APPROVED** [review]
Thanks!

**Pull request merged**